aboutus.html

```diff
@@ -0,0 +1,95 @@
+							<ul class="navalign">
+								<li><a href="collaboration.html">Collaboration</a></li>
+								<li><a href="employment.html">Employment</a></li>
+								<li><a href="intervention.html">Intervention</a></li>
+								<li><a href="skilledprofessional.html">Skilled Profesional</a></li>
+							</ul>
+						</li>
+						<li><a href="contactus.html">CONTACT</a></li>
+					</ul>
+				</nav>
+		</header>
+
+		<section>
+			<div class="mainphoto skilled">
+				<h1 class="homepageheader">ABOUT US</h1>
+			</div>
+			&nbsp;
+			<div>
+				<h1>Center on Transition to Employment for Youth with Disabilities</h1>
+					<h3>The purpose of the Center on Transition to Employment for Youth with Disabilities is to identify, research, and widely disseminate evidence-based practices that contribute to successful employment for youth with disabilities making the transition from school to adult employment. Through six discrete studies, the Center is designed to research and spotlight the strategies and circumstances that produce optimal employment and career achievement for these youth, who continue to transition from school to adult life with much lower prospects for employment success than their non-disabled peers. Thus, the Center will provide a comprehensive and contemporary view of the strategies and models of school to work transition for youth with disabilities in our country.</h3>
+						<p>Three groups of youth, who tend to have the lowest employment outcomes, are targeted for research: youth with intellectual and other developmental disabilities, youth with emotional or mental health disabilities, and minority youth with disabilities. Contribution to knowledge for ongoing academic analysis, policy development, and in-the-field practice that promote increased employment success for these youth is a key component of the Center’s mission.</p>
+						<p>The six studies include:</p>
+							<ol>
+								<li>A systematic review of promising practices for transitioning young adults with disabilities to employment.</li>
+								<li>An empirical approach to identifying promising employment-focused transition practices: Risk modeling of the National Longitudinal Transition Study (NLTS) 2.</li>
+								<li>An analysis of data from a standardized, multi-site transition to employment program serving primarily minority urban youth to identify factors that explain work outcomes and predict employment success.</li>
+								<li>A study of characteristics and perceptions of employment specialists who achieve high employment outcomes for transition-age urban and at-risk youth with disabilities.</li>
+								<li>An evaluation of the implementation and outcomes of secondary school supported employment programs for students with intellectual or development disabilities.</li>
+								<li>An implementation and evaluation of a seamless transition service model and the collaboration among participating partners.</li>
+							</ol>
+						<br>
```

Comment:
if you're trying to create white space with the `<br>` maybe you can just use `margin-bottom` on either the preceding element or `margin-top` on the following element?